src/store.js

```diff
@@ -0,0 +1,26 @@
+import thunkMiddleware from 'redux-thunk'
+import alerts from './reducers/alerts'
+import accounts from './reducers/accounts'
+import network from './reducers/network'
+import fetching from './reducers/fetching'
+import jurisdiction from './reducers/jurisdiction'
+import validators from './reducers/validators'
+import validations from './reducers/validations'
+import { createStore, combineReducers, applyMiddleware } from 'redux'
+
+const mainReducer = combineReducers({
+  alerts,
+  accounts,
+  network,
+  fetching,
+  jurisdiction,
+  validators,
+  validations,
+});
+
+const Store = createStore(
+  mainReducer,
+  applyMiddleware(thunkMiddleware)
```

Comment:
It's quite useful to add [logger middleware](https://github.com/evgenyrodionov/redux-logger) for debugging during development. Remember to add it last in the chain!

Reply:
That will be very useful! added, thanks